config/nonprod.tfvars

```diff
@@ -1,4 +1,3 @@
 accounts = [
-    { name = "plat-ops-testuser1" },
-    { name = "plat-ops-testuser2" }
+    { name = "divorce-nonprod" }
```

Comment:
please change the module to add the environment suffix

Reply:
good spot - done.